ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -52,25 +66,43 @@ public function transform($nodeGroupRole)
     }
 
     /**
-     * @param FacadeInterface             $facade
-     * @param NodeGroupRoleInterface|null $source
+     * @param FacadeInterface     $nodeRoleFacade
+     * @param GroupInterface|null $group
      *
      * @throws RoleNotFoundHttpException
-     * @return mixed
+     * @throws TransformerParameterTypeException
+     *
+     * @return null|NodeGroupRoleInterface
      */
-    public function reverseTransform(FacadeInterface $facade, $source = null)
+    public function reverseTransform(FacadeInterface $nodeRoleFacade, $group = null)
```

Comment:
Todo :

 - Create a `TransformerWithContentInterface` with the new method
 - Add the interface to the `NodeGroupRoleTransformer`
 - Check in the `GroupTransformer` if the asked transformer implements the interface